hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/LazyCreateThroughPrecompileSuite.java

```diff
@@ -204,7 +204,7 @@ HapiSpec resourceLimitExceededRevertsAllRecords() {
                                 .toArray(HapiSpecOperation[]::new)));
     }
 
-    HapiSpec autoCreationFailsWithMirrorAddress() {
+    private HapiSpec autoCreationFailsWithMirrorAddress() {
```

Comment:
Same here, no changes unless it's making a test pass.

Reply:
Agree, reverted.